docs/inspectAnnotation.md

```diff
@@ -0,0 +1,31 @@
+# Inspect Annotation
+
+Baremetalhost's(BMH) _Status_ sub-resource contain a _hardware_ key
+which contains the result of introspection which is carried out during
+BMH registration.
+
+In some circumstances it may be desirable to disable this inspection process,
+and provide data from external source.  The _Inspect Annotation_ provides some
+interfaces to enabled this.
+
+Note the `inspect.metal3.io/hardwaredetails` annotation is consumed:
```

Comment:
Could be worth mentioning here also the conditions under which the annotation will be removed (ie when successfully processed or when the status is already set)?

Reply:
Done, thanks!